components/edit-collective/actions/EmptyBalance.js

```diff
@@ -24,6 +25,68 @@ const EmptyBalance = ({ collective, LoggedInUser }) => {
 
   const closeModal = () => setModal({ ...modal, show: false, isApproved: false });
 
+  const renderSendMoneyToCollectiveSection = toCollective => {
```

Comment:
Since this basically is a React component, it might be clearer to move it outside of `EmptyBalance`, as a `SendMoneyToCollectiveSection` component. The pattern used here tends to make things difficult to read.

Reply:
@Betree : Done, I think now it's nicer. I've named it as `SendFundsToCollectiveSection` instead of `SendMoneyToCollectiveSection` as the word `Funds` seem nicer to me. Let me know if that's okay and everything looks good. 😄